.prow.yaml

```diff
@@ -133,3 +133,53 @@ presubmits:
             requests:
               memory: 1Gi
               cpu: 500m
+
+postsubmits:
+  - name: ci-push-operating-system-manager-image
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/operating-system-manager.git"
+    branches:
+      - ^master$
+      # Match on tags
+      - ^v\d+\.\d+\.\d+.*
+    labels:
+      preset-docker-push: "true"
+      preset-goproxy: "true"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/go-docker:15.1-1903-0
+          command:
+            - /bin/bash
+            - -c
+            - >-
+              set -euo pipefail &&
+              /usr/local/bin/entrypoint.sh &&
+              docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD &&
+              docker login -u $QUAY_IO_USERNAME -p $QUAY_IO_PASSWORD quay.io &&
+              make download-gocache docker-image-publish
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1Gi
+
+  - name: ci-push-operating-system-manager-upload-gocache
```

Comment:
We usually use `post-<repo-name>` prefix for postsubmits.

```suggestion
  - name: post-operating-system-manager-upload-gocache
```

Reply:
Not quite sure about the `We` notion here ;)